src/main/java/se/iths/springbootgroupproject/entities/User.java

```diff
@@ -12,9 +12,8 @@ public class User {
     private Long id;
     @Column(unique = true)
     private String userName;
-    @Lob
     @Column(name = "image")
-    private Byte[] image;
+    private String image;
```

Comment:
I get why we want the image as a URL, it's very much easier to save a URL for the picture we use on GitHub. Though this changes how I was planning to save the profile-pictures in our DB, since we need to be able to change the profile pictures. We'll have to find another solution :)
This also changes how our table looks like, since now it's a BLOB for the byteArray. We will have to use flyway to migrate this to use a VARCHAR again for the image, you can do this in your PR if you want to. Otherwise we can add another PR with updated flyway migration.

Reply:
> I get why we want the image as a URL, it's very much easier to save a URL for the picture we use on GitHub. Though this changes how I was planning to save the profile-pictures in our DB, since we need to be able to change the profile pictures. We'll have to find another solution :) This also changes how our table looks like, since now it's a BLOB for the byteArray. We will have to use flyway to migrate this to use a VARCHAR again for the image, you can do this in your PR if you want to. Otherwise we can add another PR with updated flyway migration.

Going back and forth, we thought that saving the URL and changing picture by changing the URL would be an easy way to handle profile pictures. Did not think about flyway though, good point. 
I do not think it is worth migrating because of a change that is not vital for the PR issue, that is oAuth2 login. I believe we can rollback the image type change since it is not part of this issue and it should probably be solved elsewhere (loading, saving, converting bytes or what need to be done etc) like its own issue or the profilepage-issue or such. What do you think?

Reply:
> Going back and forth, we thought that saving the URL and changing picture by changing the URL would be an easy way to handle profile pictures.

We talked about this but maybe we did not understand eachother. Because I am still not sure what you mean by this, we cannot only have the URL saved in our database. That would only work if the picture is uploaded elsewhere, and are we really then following the instructions for the project? 

> I do not think it is worth migrating because of a change that is not vital for the PR issue, that is oAuth2 login. I believe we can rollback the image type change since it is not part of this issue and it should probably be solved elsewhere (loading, saving, converting bytes or what need to be done etc) like its own issue or the profilepage-issue or such. What do you think?

I think migrating might be easier than rolling back your code. Sure it is technically not a part of the PR, but displaying the information from Github is a big part of the Demo you made. And if we want to save the profile pictures as a URL on our Users, we have to do this later anyway.

Reply:
> > Going back and forth, we thought that saving the URL and changing picture by changing the URL would be an easy way to handle profile pictures.
> 
> We talked about this but maybe we did not understand eachother. Because I am still not sure what you mean by this, we cannot only have the URL saved in our database. That would only work if the picture is uploaded elsewhere, and are we really then following the instructions for the project?
> 
Probably not right of me to only call it URL..thats why you get confused I think 😅 
With a String we can use Path or URL to point to where the picture is. The picture could be uploaded to the server to a project dir and we could save that path in our DB. Also grabbing the avatar url from github gets easy by using String instead of byte[]. We just thought that would be a nice way to save it.

Reply:
> Probably not right of me to only call it URL..thats why you get confused I think 😅 With a String we can use Path or URL to point to where the picture is. The picture could be uploaded to the server to a project dir and we could save that path in our DB. Also grabbing the avatar url from github gets easy by using String instead of byte[]. We just thought that would be a nice way to save it.

Okay that's what I thought you were talking about, but I wasn't sure anymore 😅 So we need to find a way to save the picture on the server, to then save the URL/Path in the mysql table. Sounds good. Since we want the picture to be a String, we need to make a migration. And I think it is fine if you do it in this PR.

Reply:
> > Probably not right of me to only call it URL..thats why you get confused I think 😅 With a String we can use Path or URL to point to where the picture is. The picture could be uploaded to the server to a project dir and we could save that path in our DB. Also grabbing the avatar url from github gets easy by using String instead of byte[]. We just thought that would be a nice way to save it.
> 
> Okay that's what I thought you were talking about, but I wasn't sure anymore 😅 So we need to find a way to save the picture on the server, to then save the URL/Path in the mysql table. Sounds good. Since we want the picture to be a String, we need to make a migration. And I think it is fine if you do it in this PR.

Great!